doc.ddoc

```diff
@@ -66,27 +66,54 @@ $(DIVID cssmenu, $(UL
         http://twitter.com/search?q=%23dlang, Twitter,
         http://digitalmars.com/d/dlinks.html, More Links
       )
+    $(MENU_W_SUBMENU Compilers $(AMP) Tools)
+      $(SUBMENU
+        dmd-windows.html, dmd $(NDASH) reference compiler,
+        http://gdcproject.org, gdc $(NDASH) gcc-based compiler,
+        http://wiki.dlang.org/LDC, ldc $(NDASH) LLVM-based compiler,
+        rdmd.html, rdmd $(NDASH) pseudo interpreter,
+        windbg.html, windbg $(NDASH) debugger,
```

Comment:
Using windbg is now outdated and should be expunged. On Windows, the canonical way to debug programs is now to use Visual Studio (with cv2pdb or Mago for 32-bit).